tests/gis_tests/test_measure.py

```diff
@@ -38,6 +38,11 @@ def test_init_invalid(self):
         with self.assertRaises(AttributeError):
             D(banana=100)
 
+    def test_init_invalid_area_only_units(self):
+        "Testing initialization from area only units"
```

Comment:
I know you treated `test_init_invalid` as a pattern, but this docstring is unnecessary. Please remove it.